bool sensor_1
bool sensor_2
bool sensor_3
float32 target_vel
float32 current_vel